### PR TITLE
chore(tests): updated test regex, some helper scripts

### PR DESCRIPTION
**Pull request by anshumanv**
ISSUES CLOSED: #808

**What kind of change does this PR introduce?**
Updated test regex to run all tests, added some helper scripts for running tests

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yes

**Summary**
* Updated tests regex to run all tests
* Added helper scripts to run cli and package tests separately

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by anshumanv** [comment]
> Nice one! We found out that those async which should not be there. Could you please remove them?

Ah! I forgot to test on nvm :(

Removed! 👍 💯

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -110,6 +122,18 @@ In case you are suggesting a new feature, we will match your idea with our curre
   yarn test
   ```
 
+* Run CLI tests with:
+  
+  ```bash
+  yarn test:cli`
+  ```
+
+* Run package tests with:
```
Comment (ematipico) [inline .github/CONTRIBUTING.md]: `package`  => `packages`

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -83,6 +83,18 @@ In case you are suggesting a new feature, we will match your idea with our curre
   npm run test
   ```
 
+* Run CLI tests with:
+  
+  ```bash
+  npm run test:cli
+  ```
+
+* Run package tests:
```
Comment (ematipico) [inline .github/CONTRIBUTING.md]: `package`  => `packages`

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (anshumanv) [reply .github/CONTRIBUTING.md]: Updated the entire message, looks better now I think

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (anshumanv) [reply .github/CONTRIBUTING.md]: Updated

**Comment by anshumanv** [comment]
@ematipico updated! 💯

**Pull request merged**